### PR TITLE
Move defaultRegion to validator instance

### PR DESCRIPTION
**Pull request by BoShurik**
To set default region for entire application but not in every constraint

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,8 +21,6 @@ matrix:
           env: SYMFONY_VERSION=^4.3
         - php: 7.3
           env: SYMFONY_VERSION=^5.0
-        - php: 7.4
-          env: SYMFONY_VERSION=^5.0
```
Comment (odolbeau) [inline .travis.yml:25]: Why removing this test?

**Comment by odolbeau** [comment]
My only concern regarding this PR is about the configuration.
The change you made looks like a BC Break. As we didn't release a new version since your previous PR this is currently not a problem but will it still be straightforward to add configuration for a given "component" as you did here for the validator?
For example, If I want to add a new configuration key (under `serializer` for instance) will it be easy to achieve while keeping the compatibility?
I guess it's OK but I didn't play with the `serializer` since a while.

File: .travis.yml (hunk 1 above)
Comment (BoShurik) [reply .travis.yml:25]: This case covered by third job: https://github.com/odolbeau/phone-number-bundle/pull/41/checks?check_run_id=1264722969

**Comment by BoShurik** [comment]
Yes, you are right. I'll fix this

**Review by maxhelias — APPROVED** [review]
LGTM

File: .travis.yml (hunk 1 above)
Comment (odolbeau) [reply .travis.yml:25]: :ok_hand:

**Review by odolbeau — APPROVED** [review]

**Comment by odolbeau** [comment]
@maxhelias :ok_hand:

**Comment by maxhelias** [comment]
Thanks @BoShurik for all your work on the configuration 😃

**Comment by BoShurik** [comment]
@odolbeau @maxhelias is it possible to tag new release (3.2 I suppose)?

**Comment by odolbeau** [comment]
@BoShurik Done! https://github.com/odolbeau/phone-number-bundle/releases/tag/v3.2.0

Thanks a lot for your work! :)

**Pull request merged**